### PR TITLE
1.8.97

### PR DESCRIPTION
**Pull request by Matteo-it**
v1.8.97 include:

- GitHub embed code in markdown files ([Jira task](https://arduino.atlassian.net/jira/software/projects/WEDO/boards/381?selectedIssue=WEDO-4236))
- Quickstart links in essentials ([Jira task](https://arduino.atlassian.net/jira/software/projects/WEDO/boards/381?selectedIssue=WEDO-4326))
- Add icons for software ([Jira task](https://arduino.atlassian.net/jira/software/projects/WEDO/boards/381?selectedIssue=WEDO-4286))
- Docs certifications: ([Jira task](https://arduino.atlassian.net/jira/software/projects/WEDO/boards/381?selectedIssue=WEDO-4306))

### CODE REVIEW

File: content/learn/03.programming/05.arduino-and-python/arduino-and-python.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+description: "Learn about compatibility between the popular MicroPython implementation and Arduino boards, how to set up your environment, and discover in-depth tutorials and useful links."
+tags:
   - MicroPython
   - Arduino Lab
   - OpenMV
 hero_position: 1
 ---
 
-***To download the firmware required to run MicroPython on your Arduino board, visit the [Arduino MicroPython downloads page](/micropython).***
+**_To download the firmware required to run MicroPython on your Arduino board, visit the [Arduino MicroPython downloads page](/micropython)._**
```
Comment (jhansson-ard) [inline content/learn/03.programming/05.arduino-and-python/arduino-and-python.md:12]: @Matteo-it why was this changed? This should still be three *** since this is what creates the "note"-tags.

Comment (jhansson-ard) [reply content/learn/03.programming/05.arduino-and-python/arduino-and-python.md:12]: There are other changes in this file as well where this change was implemented where it should not :(

Comment (Matteo-it) [reply content/learn/03.programming/05.arduino-and-python/arduino-and-python.md:12]: Sorry @jhansson-ard, i was testing the new CodeBlock in this page, i will replace this file with the main version soon :D

File: content/learn/03.programming/05.arduino-and-python/arduino-and-python.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
-author: 'Karl Söderby'
-title: 'MicroPython with Arduino Boards'
-description: 'Learn about compatibility between the popular MicroPython implementation and Arduino boards, how to set up your environment, and discover in-depth tutorials and useful links.'
-tags: 
+author: "Karl Söderby"
```
Comment (CloruroDiSodio) [inline content/learn/03.programming/05.arduino-and-python/arduino-and-python.md:2]: Was this file modified by the linter?

Comment (Matteo-it) [reply content/learn/03.programming/05.arduino-and-python/arduino-and-python.md]: Noe i use this file as a test, restored with the main version with the last push

**Review by CloruroDiSodio — APPROVED** [review]
